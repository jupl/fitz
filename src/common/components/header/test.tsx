import {create} from 'react-test-renderer'

test('render', async () => {
  const React = await import('react')
  const {Header} = await import('.')
  const component = create(<Header title="Title" />)
  expect(component.toJSON()).toMatchSnapshot()
})
