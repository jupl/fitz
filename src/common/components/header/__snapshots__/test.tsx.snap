// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<header
  className=""
>
  <div
    className=""
  >
    <h1>
      <a
        className=""
        href="/"
      >
        Title
      </a>
    </h1>
  </div>
</header>
`;
